cpp/cmake_modules/FindArrow.cmake

```diff
@@ -87,16 +87,19 @@ if(MSVC)
     set(ARROW_MSVC_STATIC_LIB_SUFFIX "_static")
   endif()
 
+  # Do not use LIB_PATH_SUFFIXES - DLL files are installed in "/bin"
+  # and static objects are in "/lib", so we want to search "/bin"
+  # first
   find_library(ARROW_SHARED_LIBRARIES
                NAMES arrow
                PATHS ${ARROW_HOME}
-               PATH_SUFFIXES ${LIB_PATH_SUFFIXES}
+               PATH_SUFFIXES "bin"
```

Comment:
`PATH_SUFFIXES "bin" ${LIB_PATH_SUFFIXES}` will be more portable.

Reply:
Good idea, fixed.